src/stepFunctions/asl/aslServer.ts

```diff
@@ -258,6 +270,15 @@ function triggerValidation(textDocument: TextDocument): void {
     }, validationDelayMs)
 }
 
+// sets language service depending on document language
+function getLanguageService(langId: string): LanguageService {
+    if (langId === 'asl-yaml') {
+        return aslYamlLanguageService
+    } else {
```

Comment:
Is there a chance that the `else` picks up a different language ID?

Reply:
yes, it's either `asl` or `asl-yaml`